mmseg/core/optimizers/layer_decay_optimizer_constructor.py

```diff
@@ -115,6 +138,15 @@ def add_params(self, params, module):
                 layer_id = get_num_layer_layer_wise(
                     name, self.paramwise_cfg.get('num_layers'))
                 logger.info(f'set param {name} as id {layer_id}')
+            elif decay_type == 'layer_wise_vit':
+                warnings.warn(
+                    'DeprecationWarning: Original '
+                    'LayerDecayOptimizerConstructor '
+                    'has been deprecated. Please use '
+                    'LearningRateDecayOptimizerConstructor instead, '
+                    'and set decay_type = layer_wise_vit in paramwise_cfg.')
```

Comment:
Can users directly train BEiT in this pr branch without any modification? If so, `and set decay_type = layer_wise_vit in paramwise_cfg.` seems like let users do something manually.

Reply:
Yep. I have modified the BEiT config, and compared the lr and wd parameters of the original training. After modification, they are the same as before. This change will not affect convnext and Beit training.

Reply:
Users can train Beit directly without modifying any configs.

Reply:
Users can train BEiT directly without modifying any configs.

Reply:
Yep. I have modified the BEiT config, and compared the lr and wd parameters of the original training. After modification, they are the same as before. This change will not affect ConvNeXt and BEiT training.

Reply:
OK